app/connectors/InvitationConnector.scala

```diff
@@ -49,14 +50,16 @@ class ActiveRelationshipExistsException extends AcceptInvitationException
 
 class InvitationConnectorImpl @Inject()(http: HttpClient, config: FrontendAppConfig) extends InvitationConnector with HttpResponseHelper {
 
-  override def invite(invitation: Invitation)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Int] = {
+  override def invite(invitation: Invitation)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Unit] = {
+    val nameMatchingFailedMessage = "The name and PSA Id do not match"
```

Comment:
Might be better to externalize these two strings to one central place??

Reply:
this is just used in one place and currently not re-usable